lib/fungus_toast/games/grid.ex

```diff
@@ -0,0 +1,32 @@
+defmodule FungusToast.Games.Grid do
+    alias FungusToast.Games.GridCell
+
+def create_starting_grid(number_of_rows, number_of_columns, player_ids) do
+    initial_grid_cells_map = map_of_player_id_to_random_grid_index(number_of_rows, number_of_columns, player_ids)
+
+    Enum.reduce(initial_grid_cells_map, %{}, fn {player_id, position}, map ->
+      Map.put(map, position, %GridCell{
+        index: position,
+        player_id: player_id,
+        live: true,
+        empty: false
+      })
+    end)
+  end
+
+  defp map_of_player_id_to_random_grid_index(number_of_rows, number_of_columns, player_ids) do
+    number_of_players = length(player_ids)
+    number_of_cells_to_choose_from = number_of_columns * number_of_rows / number_of_players
```

Comment:
Does this essentially divide the boards in half/thirds/quarters and only allow a player to select from one of those pieces?

Reply:
Yes -- but the player can't select - they just get a random cell in that space. I think we'll just end up having pre-configured starting locations initially (per last evening's slack conversations) -- then we don't have to deal with this.

Reply:
Oh yeah - ha! I forgot about that Slack convo 👍 (I was tired)